src/renderer/browser/util/parseURL.js

```diff
@@ -16,5 +17,11 @@ export default function parseURL(query) {
   }
 
   const tld = getTLD(trimmedQuery);
-  return new URL(`${tld}//${trimmedQuery}`);
+  const url = `${tld}//${trimmedQuery}`;
+
+  try {
+    return new URL(url);
+  } catch (err) {
+    return new URL(`https://www.google.com/search?q=${encodeURIComponent(trimmedQuery)}`);
```

Comment:
Let's change this to `https://duckduckgo.com/?q=${encodeURIComponent(trimmedQuery)}`?